gslab_scons/builders/build_tables.py

```diff
@@ -23,16 +24,40 @@ def build_tables(target, source, env):
     # Prelims
     source = misc.make_list_if_string(source)
     target = misc.make_list_if_string(target)
+
+    start_time  = misc.current_time()
     
     # Set up source file (table format)
     source_file = str(source[0])
     misc.check_code_extension(source_file, '.lyx')
 
+    # Set up input string (list of data tables)
+    input_string = ' '.join([str(i) for i in source[1:]])
+
     # Set up target file (filled table)
     target_file = str(target[0])
+    target_dir  = misc.get_directory(target_file)    
+    log_file = target_dir + '/sconscript.log'
     misc.check_code_extension(target_file, '.lyx')
```

Comment:
The log file construction should be after the target_file check (swap the two lines)